lib/firefox.js

```diff
@@ -13,7 +13,7 @@ var GARBAGE = [
   /\[JavaScript Warning: "TypeError: "[\w\d]+" is read-only"\]/,
   /JavaScript strict warning: /,
   /\#\#\#\!\!\! \[Child\]\[DispatchAsyncMessage\]/,
-  /JavaScript strict warning: resource:\/\/\/modules\/sessionstore\/SessionStore\.jsm/,
+  /JavaScript strict warning: resource:\/\/\/modules\/sessionstore\/SessionStore\.jsm/, // eslint-disable-line max-len
```

Comment:
Can these be concatenated and broken into separate lines like `/first/ + /second/` ? That might be harder to read, I'm not sure.